api/src/main/java/org/openmrs/module/aijar/api/impl/AijarServiceImpl.java

```diff
@@ -77,12 +77,12 @@ public void linkExposedInfantToMotherViaARTNumber(Person infant, String motherAR
 		// find the mother by identifier
 		List<Patient> mothers = patientService.getPatients(null, // name of the person
 				motherARTNumber, //mother ART number
-				Arrays.asList(Context.getPatientService().getPatientIdentifierTypeByUuid(PatientIdentifierTypes.HIV_CARE_NUMBER.uuid())), // ART Number Identifier type
+				Arrays.asList(Context.getPatientService().getPatientIdentifierTypeByUuid(PatientIdentifierTypes.ART_PATIENT_NUMBER.uuid())), // ART Number Identifier type
```

Comment:
@ssmusoke how is this change tied to the upgrade

Reply:
The upgrade exposed the fact that there is no motherARTNumber which matches with HIV_CARE_NUMBER

Reply:
@slubwama I have actually updated this method to search both identifier types when trying to match the mother's ART number when linking a mother to an HIV exposed infant. 

@dkayiwa thanks for finding this bug after over 3 years in active use ;-)

Reply:
@dkayiwa Yes I see that now - however I am looking at how I can search across two identifier types but it does not seem to work for a list

Reply:
@dkayiwa Seems like the patient search is broken for two identifier types - any thoughts for working around it?

Reply:
That was a bug in the platform. https://issues.openmrs.org/browse/TRUNK-5696
You can either reference platform 2.2.1-SNAPSHOT or revert the test to one identifier type.

Reply:
@dkayiwa I have upgraded to the snapshot which works since we need the program attributes features

@slubwama this now passes

Reply:
If you have the bandwidth, you can also release the 2.2.1 maintenance version, or simply mentor a community volunteer to do it.